ReactiveTask/Task.swift

```diff
@@ -544,8 +544,9 @@ public func launchTask(taskDescription: Task, standardInput: SignalProducer<NSDa
 						}
 						dispatch_group_leave(group)
 					}
-
-					task.launch()
+					
+					observer.sendNext(.Launch(task))
+					rawTask.launch()
```

Comment:
After looking at this, I think it should be acceptable to just send the `.Launch` immediately before launching the taks.